crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -501,12 +505,20 @@ pub struct AuthorityEpochTables {
 
     /// Records messages processed from other nodes. Updated when receiving a new dkg::Message
     /// via consensus.
-    pub(crate) dkg_processed_messages: DBMap<PartyId, dkg::ProcessedMessage<PkG, EncG>>,
+    pub(crate) dkg_processed_messages_v2: DBMap<PartyId, VersionedProcessedMessage>,
+    #[deprecated]
```

Comment:
i'd not mark these as deprecated until we actually remove the code that uses the old tables

Reply:
i'd rather be explicit about it, they should not be used for new code